bitser.lua

```diff
@@ -322,7 +323,7 @@ local function deserialize_value(seen)
 			local key = deserialize_value(seen)
 			v[key] = deserialize_value(seen)
 		end
-		if t == 253 then
+		if t == 253 and handleMetatables then
```

Comment:
What is your rationale for checking the metatable setting here? I'm not sure valid bitser output should error when loaded with the wrong flags.

Reply:
Indeed, will fix.